packages/base-styles/_variables.scss

```diff
@@ -127,3 +127,6 @@ $block-selected-border-width: 1px;
 $block-selected-padding: 0;
 $block-selected-child-margin: 5px;
 $block-selected-to-content: $block-edge-to-content - $block-selected-margin - $block-selected-border-width;
+
+// Block Preview
+$block-preview-height: 150px;
```

Comment:
I like the variable as it's self-documenting but I don't imagine that it will be used much elsewhere. I'd move it to `edit-site/src/components/global-styles/style.scss`.